tools/SaveModules.ps1

```diff
@@ -0,0 +1,114 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+ .Synopsis
+  Save Azure Resource Manager cmdlet modules
+
+ .Description
+  Installs all the available Azure Resource Manager cmdlet modules that have a matching major version.
+
+ .Parameter MajorVersion
+  Specifies the major version.
+
+ .Parameter Repository
+  Limit the search for "AzureRM" cmdlets in a specific repository.
+#>
+param(
+  [Parameter(Position = 0, Mandatory = $false)]
+  [string]
+  $BuildConfig = "Release",
+  [Parameter(Position = 1, Mandatory = $false)]
+  [string]
+  $Repository = "PSGallery")
+
+$AzureMajorVersion = "0"
+
+$AzureRMModules = @(
+  "AzureRM.ApiManagement",
```

Comment:
I think we should probably have a major version for each module - over time these are going to come off the original major version